src/AWS.Deploy.CLI/AWSUtilities.cs

```diff
@@ -67,12 +76,12 @@ async Task<AWSCredentials> Resolve()
 
                 try
                 {
-                    var fallbackCredentials = FallbackCredentialsFactory.GetCredentials();
+                    var fallbackCredentials = _fallbackCredentialsFactory.Create();
```

Comment:
Since the `IFallbackCredentialsFactoryFactory` is not creating a `FallbackCredentialsFactory` but instead returning `AWSCredentials` then it should be an `IAWSCredentialsFactory`.

Reply:
Don't forget to fix the variable names after making the changes